fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1639,7 +1639,7 @@ void buildPartitions(Database db, OlapTable table, List<Partition> partitions) t
             numReplicas += partition.getReplicaCount();
         }
 
-        if (partitions.size() >= 3 && numAliveBackends >= 3 && numReplicas >= numAliveBackends * 500) {
+        if (true /*partitions.size() >= 3 && numAliveBackends >= 3 && numReplicas >= numAliveBackends * 500*/) {
```

Comment:
you could use a FE config or session variable here.

Reply:
Can you please provide me a pointer to how to do that? Preferably if there's an existing example in the same file that I can C&P.

Reply:
You could refer to Config.tablet_create_timeout_second